tests/integration/test_stores.py

```diff
@@ -1,17 +1,22 @@
-import unittest
+from localstack.utils.strings import short_uid
 
 
-@unittest.mock.patch("localstack.config.ALLOW_NONSTANDARD_REGIONS", True)
-def test_nonstandard_regions(create_boto_client, monkeypatch):
+def test_nonstandard_regions(monkeypatch, create_boto_client, sqs_create_queue):
     """
     Ensure that non-standard AWS regions can be used vertically.
     """
     monkeypatch.setenv("MOTO_ALLOW_NONEXISTENT_REGION", "true")
+    monkeypatch.setattr("localstack.config.ALLOW_NONSTANDARD_REGIONS", True)
 
     # Create a resource in Moto backend
     ec2_client = create_boto_client("ec2", region_name="uranus-south-1")
-    ec2_client.create_key_pair(KeyName="foo")
+    key_name = f"k-{short_uid()}"
+    ec2_client.create_key_pair(KeyName=key_name)
+    assert ec2_client.describe_key_pairs(KeyNames=[key_name])
 
     # Create a resource in LocalStack store
     sqs_client = create_boto_client("sqs", region_name="pluto-central-2a")
-    sqs_client.create_queue(QueueName="bar")
+    queue_name = f"q-{short_uid()}"
+    sqs_client.create_queue(QueueName=queue_name)
```

Comment:
It is not possible to use the `sqs_create_queue` fixture here because the client needs to use a non-standard region

Reply:
yeah, hopefully soon gonna be a thing of the past :sunglasses: anyway